bin/API_migration_scripts/API_changes_identifier.py

```diff
@@ -0,0 +1,248 @@
+#API changes identifier
+import os
+import csv
```

Comment:
I had this an earlier comment. Why do we need `csv` instead just using `pandas`?
https://github.com/e-mission/e-mission-phone/pull/1031#discussion_r1327899392

Reply:
Also, where is the `environment.yml` with the new packages needed by this code?